Chess/src/Chess.cpp

```diff
@@ -0,0 +1,257 @@
+	#include "Chess.h"
+
+// clear the screen "cls"
+void Chess::clear() const 
```

Comment:
Instead of using a mixture of board-related code and game logic in the Chess class, you could  Board class to handle board operations and keep the game logic focused on the game itself